### PR TITLE
use --with-tcltk* configure options to point to ensure Tcl/Tk deps are picked up

### PR DESCRIPTION
**Pull request by boegel**
fix for #728

### CODE REVIEW

File: easybuild/easyblocks/p/python.py [hunk 1]
```diff
@@ -104,6 +104,26 @@ def configure_step(self):
                 line = re.sub(r"^#(\s*-L\$\(SSL\)/lib )", r"\1 -L$(SSL)/lib64 ", line)
                 sys.stdout.write(line)
 
+        tcl = get_software_root('Tcl')
+        tk = get_software_root('Tk')
+        if tcl and tk:
```
Comment (wpoely86) [inline easybuild/easyblocks/p/python.py]: maybe warning in the `else`?

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyblocks/p/python.py (hunk 1 above)
Comment (boegel) [inline easybuild/easyblocks/p/python.py]: People are free to use the system Tcl/Tk (e.g. by suing `--filter-deps`), which is fine.

**Comment by boegel** [comment]
Testing confirms that this fixes the (original) issue reported in #728.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**